implants/lib/eldritch/src/file/temp_file_impl.rs

```diff
@@ -1,44 +1,75 @@
 use anyhow::Result;
-use std::fs;
-use std::path::PathBuf;
+use std::env;
+use std::fs::File;
 use tempfile::NamedTempFile;
 
 pub fn temp_file(name: String) -> Result<String> {
```

Comment:
To change the name to optional use something like `file.find`
https://github.com/spellshift/realm/blob/79bdf000e60492727d2e8eb0a9c5a00752c27ee3/implants/lib/eldritch/src/file/mod.rs#L170C69-L170C95


`Option<String>` instead of `String`